tests/v2/validation/build.sh

```diff
@@ -4,6 +4,7 @@ set -x
 set -eu
 
 DEBUG="${DEBUG:-false}"
+CLI_VERSION="${CLI_VERSION:-v2.8.3}"
```

Comment:
can you update this to not have a default value? that way the former comment of running only if not empty is valid. This it to save time on the build

Reply:
Resolved